packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -249,6 +249,19 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
     }
   }
 
+  protected _dismissOnBlur(ev: Event) {
+    const target = ev.target as HTMLElement;
+    const clickedOutsideCallout = this._hostElement.current && !elementContains(this._hostElement.current, target);
```

Comment:
Not a must, but I would encourage this...

When I fix a bug and I write code like this, I like to provide context as well in inline comments e.g.

```
//  We implement a blur listener for the Callout component so that if a page inside an iframe uses a callout, it can be 
// dismissed when the user clicks outside of the iframe.
```

Helps everyone down the road.